packages/webapp/src/containers/Animals/Inventory/useFilteredInventory.ts

```diff
@@ -79,4 +94,10 @@ export const useFilteredInventory = (inventory: AnimalInventory[]) => {
     groupsFilter,
     locationsFilter,
   ]);
+
+  if (showOnlySelected) {
```

Comment:
So if I'm understanding correctly, the filterMatches are never used in this case? Then I think this conditional could go at the top of the hook (above the useMemo) so that the useMemo logic doesn't have to run at all. It's also not necessary to include showOnlySelected in the useMemo because when it's true, the useMemo return is irrelevant, and when it's false, I think the new conditional line is skipped anyway, right? So it behaves as before?

Reply:
Yeah thanks I moved it up and removed the excess return statement!